Dual Account Script/README.md

```diff
@@ -12,14 +12,16 @@ For more information, see [here](https://docs.cyberark.com/Product-Doc/OnlineHel
 
 This script enables fully automatic configuration for the dual account feature.
 
+**_Notice:_** In Privilege Cloud Shared Services environment, only password authentication method is supported.
+
```

Comment:
only password authentication method is supported
what does it mean? why only this method?

Reply:
the only authentication that is supported is password (not email, sms and etc)